ingestion/src/metadata/profiler/interface/profiler_interface.py

```diff
@@ -204,6 +350,35 @@ def get_profile_sample_config(
 
         return None
 
+    @staticmethod
+    def get_sample_data_count_config(
+        entity: Table,
+        schema_profiler_config: Optional[DatabaseSchemaProfilerConfig],
+        database_profiler_config: Optional[DatabaseProfilerConfig],
+        entity_config: Optional[TableConfig],
+        source_config: DatabaseServiceProfilerPipeline,
+    ) -> Optional[int]:
+        """_summary_
+        Args:
+            entity_config (Optional[TableConfig]): table config object from yaml/json file
+            source_config DatabaseServiceProfilerPipeline: profiler pipeline details
+        Returns:
+            Optional[int]: int
+        """
+        if entity_config and entity_config.sampleDataCount:
```

Comment:
nit, instead of having 4 if-else, we can do a list of objects and iterate over them? Might be easier to understand what we try to achieve